source/style-guide/style/tasks.txt

```diff
@@ -58,6 +55,20 @@ especially if it will take a long time.
     informational topic or in a topic that introduces the task as part
     of a larger process or user goal.
 
+Considerations
+~~~~~~~~~~~~~~
+
+If the task has items that the user should acknowledge or contemplate
+before taking action, describe them in a "Considerations" section that
+precedes any prerequisites or the procedure. You could include the
+following information:
+
+- Descriptions of variations, options, or scopes
+- Hyperlinks to information that prepares the user for any choices they
+  must make
+- Impacts on other areas of the product or later tasks they might want 
+  to complete
+
 Prerequisites
 ~~~~~~~~~~~~~
 
```

Comment:
**suggestion:** Highlight *before* on [line 75](https://github.com/mongodb/docs-meta/pull/47/files#diff-3a63be05cb65610741007b5d0089a46bR75).

Reply:
**suggestion:** Under **Procedures**, consider referencing `step` files. This may need to be added once the markup guide is complete.

Reply:
Done and done. We'll need to add a link once the markup guide exists.